Web app build fails with `@trezor/connect-web@9.2.0`

**Describe the bug**
Updating `@trezor/connect-web` to version `9.2.0` results in build failure due to failed module resolution. Namely:
```
 ⨯ ./node_modules/@trezor/connect-web/lib/channels/serviceworker-window.js:4:19
Module not found: Can't resolve '@trezor/connect-common/src/messageChannel/abstract'
```

Version 9.1.6 worked without problems.

More info as well as steps to reproduce can be found in [this Solana wallet adapter issue](https://github.com/anza-xyz/wallet-adapter/issues/918).

In addition to the repo linked in the Wallet Adapter issue I was also able to reproduce the issue when updating `@trezor/connect-web` to `9.2.0` in [one of my own test repos](https://github.com/gabrielKerekes/solana-connect-test) and in NuFi wallet as well.

I feel like the issue might stem from changes introduced here - https://github.com/trezor/trezor-suite/commit/7febd10cf0aab51090190b7e8a4810e9562ddbc3#diff-c34fda44918aae434921e87b60b832f12afedea27dac76ba67b693f779099d6dL21.

I tried reverting these changes locally and `yarn link`ing the package to NuFi and NuFi build suddenly passed. I tried introducing the changes again and re-linking to properly isolate the issue, but I wasn't able to re-link for some reason 🙈

--- a/packages/connect-common/src/index.ts
+++ b/packages/connect-common/src/index.ts
@@ -1,3 +1,3 @@
 export * from './storage';
-
+export * from './messageChannel/abstract';
 export * from './systemInfo';

--- a/packages/connect-web/src/channels/serviceworker-window.ts
+++ b/packages/connect-web/src/channels/serviceworker-window.ts
@@ -2,7 +2,7 @@ import {
     AbstractMessageChannel,
     AbstractMessageChannelConstructorParams,
     Message,
-} from '@trezor/connect-common/src/messageChannel/abstract';
+} from '@trezor/connect-common';
 
 /**
  * Communication channel between:

--- a/packages/connect-web/src/channels/window-serviceworker.ts
+++ b/packages/connect-web/src/channels/window-serviceworker.ts
@@ -2,7 +2,7 @@ import {
     AbstractMessageChannel,
     AbstractMessageChannelConstructorParams,
     Message,
-} from '@trezor/connect-common/src/messageChannel/abstract';
+} from '@trezor/connect-common';
 
 /**
  * Communication channel between:

--- a/packages/connect-web/src/channels/window-window.ts
+++ b/packages/connect-web/src/channels/window-window.ts
@@ -2,7 +2,7 @@ import {
     AbstractMessageChannel,
     AbstractMessageChannelConstructorParams,
     Message,
-} from '@trezor/connect-common/src/messageChannel/abstract';
+} from '@trezor/connect-common';
 
 /**
  * Communication channel between:

